storage/integration_test.go

```diff
@@ -2807,6 +2809,13 @@ func TestIntegration_RequesterPays(t *testing.T) {
 	}
 	defer h.mustDeleteBucket(requesterPaysBucket)
 
+	// Create a second requester-pays bucket to avoid rate limits.
+	h.mustCreate(requesterPaysBucket2, mainProjectID, &BucketAttrs{RequesterPays: true})
+	if err := requesterPaysBucket2.ACL().Set(ctx, ACLEntity("user-"+otherUserEmail), RoleOwner); err != nil {
```

Comment:
nit: inline err

Reply:
not sure what you mean here

Reply:
Me neither! lol. Maybe I meant to comment on L2915?

Reply:
okay, I inlined that err :)